modules/virt-deleting-template-web.adoc

```diff
@@ -10,7 +10,7 @@ Deleting a virtual machine template permanently removes it from the cluster.
 .Procedure
 
 . In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
-. Click the *Virtual Machine Templates* tab.
+. Click the *Templates* tab.
 . You can delete the virtual machine template from this pane, which makes it
```

Comment:
I understand these are not changes you made in this PR but this sentence is quite wordy and can be restructured to say - Choose the appropriate method to delete a virtual machine template:'

Reply:
Cleaned up this content. Please review.